chainer/optimizer.py

```diff
@@ -675,10 +730,14 @@ class WeightDecay(object):
 
     Attributes:
         rate (float): Coefficient for the weight decay.
+        timing (string): Specifies when this hook should be called by the
+                         Optimizer/UpdateRule. Valid values are 'pre'
+                         (before any updates) and 'post' (after any updates).
```

Comment:
Could you write that `timing` is added from v4.0.0? You can use `versionadded` markup (See usage [here](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded))

Reply:
Ok, I added it. Please see if I understood the syntax correctly.